src/aggregator/client/config.go

```diff
@@ -207,6 +207,27 @@ func (c *Configuration) NewClientOptions(
 	return opts, nil
 }
 
+// TLSConfiguration contains the TLS configuration
+type TLSConfiguration struct {
+	TLSEnabled         bool   `yaml:"tlsEnabled"`
```

Comment:
should this just be `Enabled` ? it's already in a struct called TLSConfiguration in the go code, and in the yaml it's under the `tls` stanza

Reply:
You're right, `TLS` is redundant.